package.json

```diff
@@ -17,5 +17,9 @@
   },
   "dependencies": {
     "rxjs": "^5.0.2"
+  },
+  "devDependencies": {
+    "react-native": "^0.43.2",
```

Comment:
This is to include react-native libs to the visual studio solution. This is not really needed, but for easier development process, that's better :)